content/en/docs/File Format/implementationstatus.md

```diff
@@ -45,66 +45,66 @@ Implementations:
 | ----------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| PLAIN                                     |  ✅   |  ✅   |       |       |  ✅   |
-| PLAIN_DICTIONARY                          |  ✅   |  ✅   |       |       |  ✅   |
-| RLE_DICTIONARY                            |  ✅   |  ✅   |       |       |  ✅   |
-| RLE                                       |  ✅   |  ✅   |       |       |  ✅   |
-| BIT_PACKED (deprecated)                   |  ✅   |  ✅   |       |       |  (R)  |
-| DELTA_BINARY_PACKED                       |  ✅   |  ✅   |       |       |  ✅   |
-| DELTA_LENGTH_BYTE_ARRAY                   |  ✅   |  ✅   |       |       |  ✅   |
-| DELTA_BYTE_ARRAY                          |  ✅   |  ✅   |       |       |  ✅   |
-| BYTE_STREAM_SPLIT                         |  ✅   |  ✅   |       |       |  ✅   |
+| PLAIN                                     |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| PLAIN_DICTIONARY                          |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| RLE_DICTIONARY                            |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| RLE                                       |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| BIT_PACKED (deprecated)                   |  ✅   |  ✅   |       |  ❌   |  (R)  |
+| DELTA_BINARY_PACKED                       |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| DELTA_LENGTH_BYTE_ARRAY                   |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| DELTA_BYTE_ARRAY                          |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| BYTE_STREAM_SPLIT                         |  ✅   |  ✅   |       |  ✅   |  ✅   |
 
 ### Compressions
 
 | Compression                               | C++   | Java  | Go    | Rust  | cuDF  |
 | ----------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| UNCOMPRESSED                              |  ✅   |  ✅   |       |       |  ✅   |
-| BROTLI                                    |  ✅   |  ✅   |       |       |  (R)  |
-| GZIP                                      |  ✅   |  ✅   |       |       |  (R)  |
-| LZ4 (deprecated)                          |  ✅   |  ❌   |       |       |  ❌   |
-| LZ4_RAW                                   |  ✅   |  ✅   |       |       |  ✅   |
-| LZO                                       |  ❌   |  ❌   |       |       |  ❌   |
-| SNAPPY                                    |  ✅   |  ✅   |       |       |  ✅   |
-| ZSTD                                      |  ✅   |  ✅   |       |       |  ✅   |
+| UNCOMPRESSED                              |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| BROTLI                                    |  ✅   |  ✅   |       |  ✅   |  (R)  |
+| GZIP                                      |  ✅   |  ✅   |       |  ✅   |  (R)  |
+| LZ4 (deprecated)                          |  ✅   |  ❌   |       |  ✅   |  ❌   |
+| LZ4_RAW                                   |  ✅   |  ✅   |       |  ✅   |  ✅   |
+| LZO                                       |  ❌   |  ❌   |       |  ❌   |  ❌   |
```

Comment:
https://docs.rs/parquet/latest/parquet/basic/enum.Compression.html claims to support LZO
 
However I did some more digging and I agree that LZO does not appear to be supported  https://github.com/apache/arrow-rs/blob/7781bc2170c84ada387901e09b2cdfe4235c3570/parquet/src/compression.rs#L195-L194